doc/src/guides/solving/solving-guidance.md

```diff
@@ -0,0 +1,245 @@
+>>> solve(eq, x, dict=True)
+[{x: 2**(sqrt(2)/2)}]
+```
+
+but if you use the inexact equation `eq = x**1.4142135623730951 - 2`, SymPy will
+not return a result despite attempting for a long time. 
+
+## Options That Can Speed up {func}`~.solve`
+
+### Include Solutions Making Any Denominator Zero by Using `check=False`
+
+Normally, {func}`~.solve` checks whether any solutions make any denominator
+zero, and automatically excludes them. If you want to include those solutions,
+and speed up {func}`~.solve` (at the risk of obtaining invalid solutions), set
+`check=False`:
+
+```py
+>>> from sympy import Symbol, sin, solve
+>>> x = Symbol("x")
+>>> solve(sin(x)/x, x, dict=True) # 0 is excluded
+[{x: pi}]
+>>> solve(sin(x)/x, x, dict=True, check=False) # 0 is not excluded
+[{x: 0}, {x: pi}]
+```
+
+### Do Not Simplify Solutions by Using `simplify=False`
+
+Normally, {func}`~.solve` simplifies many results before returning them and (if
+`check` is not False) uses the general {func}`~sympy.simplify.simplify.simplify`
+function on the solutions and the expression obtained when they are substituted
+into the function which should be zero. If you do not want the solutions
+simplified, and want to speed up {func}`~.solve`, use `simplify=False`.
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x, y
+>>> expr = x**2 - (y**5 - 3*y**3 + y**2 - 3)
+>>> solve(expr, x, dict=True)
+[{x: -sqrt(y**5 - 3*y**3 + y**2 - 3)}, {x: sqrt(y**5 - 3*y**3 + y**2 - 3)}]
+>>> solve(expr, x, dict=True, simplify=False)
+[{x: -sqrt((y + 1)*(y**2 - 3)*(y**2 - y + 1))}, {x: sqrt((y + 1)*(y**2 - 3)*(y**2 - y + 1))}]
+```
+
+## Parse a String Representing the Equation
+
+If you are creating the expression yourself, we [against using string parsing to
+create expressions](
+https://github.com/sympy/sympy/wiki/Idioms-and-Antipatterns#strings-as-input).
+But if you are programmatically reading in a string, this approach is
+convenient.
+
+You can parse a string representing the equation into a form that SymPy can
+understand (for example, {any}`Eq() <sympy.core.relational.Eq>` form), then
+apply solve the parsed expression. Parsing an equation from a string requires
+you to use {func}`transformations <sympy.parsing.sympy_parser.parse_expr>` for
+SymPy to
+- interpret equals signs
+- create symbols from your variables
+-  use more mathematical (rather than standard Python) notation, for example the
+exponent operator can be parsed from `^` rather than having to use Python's
+`**`.
+
+To extract the solutions, you can iterate through the list of dictionaries:  
+    
+```py
+>>> from sympy import parse_expr, solve, solveset
+>>> from sympy.abc import x
+>>> expr = "x^2 = y"
+>>> parsed = parse_expr(expr, transformations="all")
+>>> print(parsed)
+Eq(x**2, y)
+>>> solutions = solve(parsed, x, dict=True)
+>>> print(solutions)
+[{x: -sqrt(y)}, {x: sqrt(y)}]
+>>> for solution in solutions:
+...     for key, val in solution.items():
+...         print(val)
+-sqrt(y)
+sqrt(y)
```

Comment:
```suggestion
>>> [s[x] for s in solutions]
[-sqrt(y), sqrt(y)]
```

Reply:
This seems to be a little out of place. Perhaps this should go up right after the suggestion about using `dict=True`.

Reply:
Thanks. This was a result of rearranging content. I realized the `dict=True` guidance was in another page ([Solve An Equation Algebraically](https://docs.sympy.org/dev/guides/solving/solve-equation-algebraically.html)) so I moved that to Solving Guidance and then was able to follow your suggestion.